examples/workspaces/iam.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_iam_role" "workspaces-default" {
+  name               = "workspaces_DefaultRole"
+  assume_role_policy = data.aws_iam_policy_document.workspaces.json
```

Comment:
This file needs to be updated for Terraform 0.11 compatibility (for a short while longer) or ignored in `,github/workflows/examples.yml` 👍 

```
===> Initializing Example: ./examples/workspaces <===

/home/runner/work/_temp/6f910e32-7342-42d2-b4bc-e4bd80c80533/terraform-bin init
There are some problems with the configuration, described below.

The Terraform configuration must be valid before initialization so that
Terraform can determine which modules and providers need to be installed.

Error: Error parsing /home/runner/work/terraform-provider-aws/terraform-provider-aws/examples/workspaces/iam.tf: At 3:24: Unknown token: 3:24 IDENT data.aws_iam_policy_document.workspaces.json
```